src/features/amUI/packagePoaDetailsPage/PackagePoaDetailsHeader.module.css

```
@@ -1,37 +1,36 @@
-  height: 2.6rem;
-  width: 2.6rem;
+  height: var(--ds-size-8);
+  width: var(--ds-size-8);
   margin: auto;
+  flex: 0 0 var(--ds-size-8);
 }
 
 .pageHeading {
-  grid-area: heading;
   max-width: 100%;
   word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)